modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsExtraInfoFieldMerge.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Configure Behavior for paragraphs.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "paragraphs_extra_info_field_merge"
+ * )
+ */
+class ParagraphsExtraInfoFieldMerge extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    // Merging the data into one field.
+    $field_uaqs_short_title = $row->getSourceProperty('field_uaqs_short_title');
+    $field_uaqs_body = $row->getSourceProperty('field_uaqs_body');
+    $field_uaqs_link = $row->getSourceProperty('field_uaqs_link');
+    $value['uaqs_text'] = '<div class="border-thick border-top border-azurite">
+    <div class="border card-body">
+       <h3>More information</h3>';
+    if ($field_uaqs_link[0]['url']) {
+      $value['uaqs_text'] .= '<a href="' . $field_uaqs_link[0]['url'] . '" class="{Custom link class}">' . $field_uaqs_link[0]['title'] . '</a>';
```

Comment:
The `{Custom link class}` value should actually be the actual field value.
```suggestion
      $value['uaqs_text'] .= '<a href="' . $field_uaqs_link[0]['url'] . '" class="' . $field_uaqs_link[0]['attributes']['class'] . '">' . $field_uaqs_link[0]['title'] . '</a>';
```